sockpuppet/reflex.py

```diff
@@ -36,7 +38,7 @@ def get_context_data(self, *args, **kwargs):
         view.request = self.request
         try:
             view.kwargs = resolved.kwargs
-            context = view.get_context_data()
+            context = view._patched_get_context_data()
```

Comment:
If you manage to fix so that we can use `get_context_data` here I think this PR will be usable. Because that's the issue where we get an infinite recursion here.